README.md

```diff
@@ -33,6 +33,12 @@ There are many ways to contribute to HEIR:
         to alert us of papers that should be investigated and incorporated into
         HEIR.
 
+## Building with CMake
+HEIR supports building with CMake with exception of building Yosys pass, tests, and examples at the time of writing.
+```bash
+$ cmake -S ./ -B builddir -DCMAKE_PREFIX_PATH=/usr/local/lib/cmake -DMLIR_CMAKE_DIR=/usr/local/lib/cmake/mlir/ -DLLVM_CMAKE_DIR=/usr/local/lib/cmake/llvm/
+```
+
```

Comment:
The other build documentation is in docs/content/docs/getting_started.md, and that's where I'd put the CMake documentation, too. However, I'd actually hold off on that a bit, until the CMake stuff is more mature.
```suggestion
```